modules/packages/Sort.chpl

```diff
@@ -1233,52 +1233,62 @@ module QuickSort {
     import Sort.InsertionSort;
 
     // grab obvious indices
-    const lo = start,
-          hi = end,
-          mid = lo + (hi-lo+1)/2;
-    var piv = mid;
-
-    if hi - lo < 0 { // minlen {
-      // base case -- use insertion sort
-      InsertionSort.insertionSortMoveElts(Data, comparator=comparator, lo, hi);
-      return;
-    } else if hi <= lo {
-      // nothing to sort
-      return;
-    }
+    var lo = start,
+          hi = end;
```

Comment:
I think we need to remove two leading spaces on this line so that the 'lo' and 'hi' are in the same columns